page-faq.php

```diff
@@ -68,6 +68,9 @@
 <p>Many people prefer to have more control over who gets to see the photos they send to their spouse or friends, who gets to read their bank account statements or determine their political allegiance. Or know where they are - as mobile phones track your location pretty much continuously...</p>
 <p>This is where ownCloud and similar technologies come in. ownCloud offers you the option to take back your data. Once you install it on a server (or let somebody do that for you! <a href="https://owncloud.org/blog/dont-be-caught-naked-in-the-cloud-decentralization-protects-our-data/" target="_blank">Decentralization is an important tool to protect privacy</a>) you can access your private data in an easy to use web interface or synchronize it with your devices like Android and iPhones. You can store your contacts and calendar as well as files on ownCloud and, through the many ownCloud apps, use it to store passwords, play music and movies and so on. It also allows you to securely share with other people and collaborate on documents. Learn more <a href="/features">about ownCloud features here</a>.</p>
 <p>All this is free, as ownCloud is available under an '<em>open source license</em>', which gives you the right to examine, share <strong>and</strong> modify it. ownCloud is developed by an <a href="/contact">international community</a> of both paid and volunteer contributors and <a href="/contribute">you can get involved, too</a>.</p>
+<h3>Things ownCloud does not do</h3>
+<p>Note that ownCloud is NOT a backup tool: the desktop client, for example, will synchronize the contents of a local folder with the server. That works for file modifications and additions, but also deletions. If you delete a file on the server, it gets deleted on the desktop you installed the client on! ownCloud does have a <strong>trasn</strong> and <strong>Versions</strong> support, which means deleted or modified files can be brought back, but this can be disabled and depending on the space usage settings, these older files do get deleted over time. You should always backup your ownCloud server!</p>
+<p>Similarly, note that External Storage and the Server to Server sharing which ownCloud supports, connects ownCloud to an external storage like FTP or Samba, or to another ownCloud server. Files will show up in your web interface and can be synced with the client. However, they are <strong>not replicated</strong> on your ownCloud server! They are merely fetched whenever you request them in the web interface. Once the external storage disappears or other ownCloud server retracts your access, you can no longer access the files! Make a copy if you want to have them on your own ownCloud server.</p>
```

Comment:
"it gets deleted on the desktop you installed the client on!"
maybe put even more, like:
"it gets deleted on all devices that you installed the client on!"
and should it also spell out the reverse by adding a sentence like:
"If you delete a file on the client, it gets deleted on the server and then deleted on other clients that sync to that were syncing that file!"
Maybe that makes the whole thing too long, and people will not read it?